swiss-ng-api/berlin-group-documentation/current.yaml

```diff
@@ -11427,6 +11708,31 @@ components:
               $ref: "#/components/examples/pain.002.001_SCT_singleTransaction"
 
 
+    OK_202_PaymentInitiationStatus:
+      description: OK
+      
+      headers: 
+        X-Request-ID:
+          $ref: "#/components/headers/X-Request-ID"
+      
+      content:
+        application/json:
+          schema:
+            $ref: "#/components/schemas/paymentInitiationStatusResponse-202_json"
+          examples:
+            simple:
+              $ref: "#/components/examples/paymentInitiationStatusResponse_json_Simple"
+            extended:
+              $ref: "#/components/examples/paymentInitiationStatusResponse_json_Extended"
+        
+        application/xml:
+          schema:
+            $ref: "#/components/schemas/paymentInitiationStatusResponse-200_xml"
+          examples: 
+            "Payment Initiation Status Response Body XML for SCT":
+              $ref: "#/components/examples/pain.002.001_SCT_singleTransaction"
+
+
```

Comment:
Relevant Change: OK_202_PaymentInitiationStatus is new
Adoption approach: Follow the change